.changeset/blue-jars-raise.md

```diff
@@ -0,0 +1,6 @@
+---
+'@rsbuild/plugin-image-compress': patch
+---
+
+fix(rsbuild): plugin-image-compress can not work with filename query
+fix(rsbuild): plugin-image-compress 在 filename 包含 query 时不生效
```

Comment:
No need for Chinese message in Rsbuild repo~

Reply:
international